### ISSUE TITLE
Plantuml Diagrams Display in Preview Not on Saved Page

### ISSUE THREAD

**Issue by tjbennett** [Requarks/wiki#2022]
**Question**
I am trying to use a local plantuml server to generate sequence diagrams in a markdown page

**Host Info (please complete the following information):**
OS: Ubuntu 18.04
Wiki.js version: 2.4.105
Database engine: postgres 10+190ubuntu0.1

Applying delimiters provides a diagram in preview mode but the save and subsequent display of the page does not show a diagram.

![previewDia](https://user-images.githubusercontent.com/375072/84094723-75523b00-a9cb-11ea-8898-b02aaff0c81d.png)
![savedNoSeqDia](https://user-images.githubusercontent.com/375072/84094732-797e5880-a9cb-11ea-90df-11dd4172ee75.png)

**Comment by Marco77577** [comment]
The same is happening on my instance of the Wiki. It seems that the new opening and closing tags are handled the same way as normal code by the saved page.

**Comment by NGPixel** [comment]
Is PlantUML enabled in Admin > Rendering?

**Comment by Marco77577** [comment]
Yes, rendering is on. However, the wrong opening and closing tags were entered (still the former ones).
Though, even after changing them, the problem remains.

**Comment by victor1234** [comment]
Have the same problem
